definitions/npm/bluebird_v3.x.x/flow_v0.32.x-/bluebird_v3.x.x.js

```diff
@@ -142,6 +142,7 @@ declare class Bluebird$Promise<R> {
   each<T, U>(iterator: (item: T, index: number, arrayLength: number) => Bluebird$Promisable<U>): Bluebird$Promise<Array<T>>;
   asCallback<T>(callback: (error: ?any, value?: T) => any, options?: Bluebird$SpreadOption): void;
   return<T>(value: T): Bluebird$Promise<T>;
+  spread<T>(...args: Array<T>): Bluebird$Promise<T>;
```

Comment:
Okay I double checked it myself... I think the better return type of this function would be `Bluebird$Promise<*>` ... otherwise we would force the user to return the same type `T` as the input type... which is not what bluebird wants, right? `*` will take whatever type is returned in the `then` body... this is definitely better than `any` :-)